### PR TITLE
feature: 使用流程模板新建任务时，返回未引用的全局变量 #36

### PR DESCRIPTION
**Pull request by pagezhou**
- 新功能
    - 使用流程模板新建任务时，返回未引用的全局变量

### CODE REVIEW

File: gcloud/tests/taskflow3/test_api.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+class APITest(TestCase):
+
+    def setUp(self):
+        self.PREVIEW_TASK_TREE_URL = '/taskflow/api/preview_task_tree/{biz_cc_id}/'
+        self.client = Client()
+
+    @mock.patch(TASKTEMPLATE_GET, MagicMock(return_value=MockBaseTemplate(id=1, pipeline_tree=TEST_PIPELINE_TREE)))
+    @mock.patch('gcloud.taskflow3.api.JsonResponse', MockJsonResponse())
+    def test_preview_task_tree(self):
```
Comment (homholueng) [inline gcloud/tests/taskflow3/test_api.py]: test_preview_task_tree__xxxxx

xxxxx 处说明测试点。

另外，最好加上边界点测试：没有未引用参数和全都是未引用参数的情况。

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-sops?pullRequest=48) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-sops?pullRequest=48) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-sops?pullRequest=48) it.</sub>

File: gcloud/tests/taskflow3/test_api.py [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+from pipeline.utils.uniqid import node_uniqid
+
+from gcloud.tests.mock import *  # noqa
+from gcloud.tests.mock_settings import *  # noqa
+from gcloud.taskflow3 import api
+
+
+TEST_BIZ_CC_ID = '2'  # do not change this to non number
+TEST_ID_LIST = [node_uniqid() for i in xrange(10)]
```
Comment (homholueng) [inline gcloud/tests/taskflow3/test_api.py]: 改成 `range(10)`，Python 3 后没有 xrange

File: gcloud/tests/taskflow3/test_api.py [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+
+class APITest(TestCase):
+
+    def setUp(self):
+        self.PREVIEW_TASK_TREE_URL = '/taskflow/api/preview_task_tree/{biz_cc_id}/'
+        self.client = Client()
+
+    @mock.patch('gcloud.taskflow3.api.JsonResponse', MockJsonResponse())
+    def test_preview_task_tree_constants_not_referred(self):
```
Comment (homholueng) [inline gcloud/tests/taskflow3/test_api.py]: `test_preview_task_tree_constants_not_referred` to `test_preview_task_tree__constants_not_referred`

**Pull request merged**